consensus/state.go

```diff
@@ -1268,7 +1268,7 @@ func (cs *State) enterPrecommit(height int64, round int32) {
 		return
 	}
 
-	logger.Info(fmt.Sprintf("enterPrecommit(%v/%v). Current: %v/%v/%v", height, round, cs.Height, cs.Round, cs.Step))
+	logger.Debug(fmt.Sprintf("enterPrecommit(%v/%v). Current: %v/%v/%v", height, round, cs.Height, cs.Round, cs.Step))
```

Comment:
What's your thinking behind having all the `enterPrevote` logs as INFO and `enterPrecommit` as DEBUG?

Reply:
I missed `enterPrevote`. I'd downgrade that to DEBUG as well.